src/api/app/models/staging/staging_project.rb

```diff
@@ -9,6 +9,18 @@ class StagingProject < Project
     after_destroy :update_staging_workflow_on_backend
     before_create :add_managers_group
 
+    def copy(new_project_name)
+      new_project = deep_clone(
+        include: [:flags, { repositories: [:path_elements, :repository_architectures, :download_repositories] }], skip_missing_associations: true
```

Comment:
you can't use deep_clone for repositories either (assuming this is for real life staging projects).

Spot the Bs in https://build.opensuse.org/projects/openSUSE:Leap:15.1:Staging:B/meta

Reply:
I'm not saying it might, I'm saying it will.

Reply:
No, you are saying it could:wink:

I am having a look at it.